pulsar-broker/src/test/java/org/apache/pulsar/broker/service/TransactionMarkerDeleteTest.java

```diff
@@ -77,7 +81,7 @@ public void testTransactionMarkerDelete() throws Exception {
                 managedLedger.openCursor("test"), false);
 
         Position position1 = managedLedger.addEntry("test".getBytes());
-        managedLedger.addEntry(Markers
+        Position position2 = managedLedger.addEntry(Markers
```

Comment:
can we simulate a sequence with multiple transactions markers:
- send
- commit
- send
- commit
...
and verify that there is no hole ?


also
- send (with tx)
- commit
- commit
- commit
- send (no tx)
....